_referendum_opposing/oakland/2016-11-08/d.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
You can remove this file. The gulp task just shoves all the referendums into the Nov 2016 election, but this is actually on the June '18 ballot. I think I'm working on a fix on another branch and will add them in the right place in that PR.

This is also the case for `tbd1` and `tbd2`, which should be on the Nov '18 ballot. All of these will all appear under `_referendums`, `_referendum_opposing`, and `_referendum_supporting`. So, 9 files in total to be removed... sorry :grimacing:

Reply:
Ah, I added election guessing logic in #62